### ISSUE TITLE
Anaconda Windows10 ImportError PIL

### ISSUE THREAD

**Issue by magnunor** [conda-forge/hyperspy-feedstock#28]
Anaconda install: Anaconda3-5.3.1-Windows-x86_64

Installing HyperSpy in Anaconda (`conda install hyperspy -c conda-forge`), and then importing it in a Jupyter Notebook leads to an ImportError:

```python
~\Anaconda3\lib\site-packages\skimage\io\manage_plugins.py in <module>()
     26 from glob import glob
     27
---> 28 from .collection import imread_collection_wrapper
     29
     30
 
~\Anaconda3\lib\site-packages\skimage\io\collection.py in <module>()
     10 import numpy as np
     11 import six
---> 12 from PIL import Image
     13
     14 from ..external.tifffile import TiffFile
 
~\Anaconda3\lib\site-packages\PIL\Image.py in <module>()
     62     # Also note that Image.core is not a publicly documented interface,
     63     # and should be considered private and subject to change.
---> 64     from . import _imaging as core
     65     if __version__ != getattr(core, 'PILLOW_VERSION', None):
     66         raise ImportError("The _imaging extension was built for another "
 
ImportError: DLL load failed: The specified module could not be found.
```

**Comment by ericpre** [comment]
This should be fixed in #27 by setting pillow >5.2 at runtime for windows.

**Comment by magnunor** [comment]
Fixed now with the conda forge HyperSpy files having been updated!
